README.md

```diff
@@ -74,33 +74,39 @@ You'll need a locally running Docker and Docker Compose:
 
 ### Quickstart
 
+Modify or create `.env.local`. This config uses the database container defaults.
+```
+DATABASE_URL=mysql://root:@database:3306/wsexport?serverVersion=5.7
+```
+
+Make sure you cd into `./docker`
+
+```bash
+cd ./docker 
+```
+
 Run the following command to add your user ID and group ID to your `.env` file:
 
 ```bash
-echo "WS_DOCKER_PORT=8888
-WS_DOCKER_UID=$(id -u)
-WS_DOCKER_GID=$(id -g)" >> .env.local
+echo "WS_DOCKER_UID=$(id -u)
+WS_DOCKER_GID=$(id -g)" >> ./.env
```

Comment:
`.env` is in version control and it gets changed with this command

Reply:
`./docker/.env` is part of gitignore, see https://github.com/wsexport/tool/pull/246/files?file-filters%5B%5D=dotfile#diff-a084b794bc0759e7a6b77810e01874f2R6

Did you run it inside `./docker`? Maybe I need to tweak the instructions

Reply:
Yup, I ran it in the root directory and it modified that .env file. Clearly I wasn't reading the instructions closely enough, hehe! Sorry, please disregard